### PR TITLE
PMM-3905 Port upstream changes.

### PR DESCRIPTION
**Pull request by DexterHD**
### Questions:

- PG_EXPORTER_WEB_LISTEN_ADDRESS was changed from 9184 to 9187. Which should we use?

### CODE REVIEW

**Comment by it-percona** [comment]
[CLA assistant check](https://cla-assistant.percona.com:443/percona/postgres_exporter?pullRequest=9) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.percona.com:443/percona/postgres_exporter?pullRequest=9) before we can accept your contribution.<br/>**1** out of **20** committers have signed the CLA.<br/><br/>:white_check_mark: AlekSi<br/>:x: tshohe<br/>:x: samuelb<br/>:x: robsonpeixoto<br/>:x: Danie<br/>:x: gsstark<br/>:x: sylr<br/>:x: bitglue<br/>:x: wrouesnel<br/>:x: regeda<br/>:x: snarlistic<br/>:x: asherf<br/>:x: NM-SC<br/>:x: michaelkaye<br/>:x: WoLpH<br/>:x: miguelHx<br/>:x: JanisOrlovs<br/>:x: sakishrist<br/>:x: rkojedzinszky<br/>:x: DexterHD<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.percona.com:443/check/percona/postgres_exporter?pullRequest=9) it.</sub>

File: cmd/postgres_exporter/postgres_exporter.go [hunk 1]
```diff
@@ -4,56 +4,44 @@ import (
...
-	)
-	queriesPath = flag.String(
-		"extend.query-path", getStringEnv("PG_EXPORTER_EXTEND_QUERY_PATH", ""),
-		"Path to custom queries to run.",
-	)
-	onlyDumpMaps = flag.Bool(
-		"dumpmaps", false,
-		"Do not run, simply dump the maps.",
-	)
+	listenAddress          = kingpin.Flag("web.listen-address", "Address to listen on for web interface and telemetry.").Default(":9187").Envar("PG_EXPORTER_WEB_LISTEN_ADDRESS").String()
```
Comment (AlekSi) [inline cmd/postgres_exporter/postgres_exporter.go]: That would require changes in pmm-managed's `postgresExporterConfig`.

Comment (DexterHD) [reply cmd/postgres_exporter/postgres_exporter.go]: Oh. That's why it doesn't work... I'll fix that.

**Comment by AlekSi** [comment]
> PG_EXPORTER_WEB_LISTEN_ADDRESS was changed from 9184 to 9187. Which should we use?

Upstream's – for easier merging in the future.

**Pull request merged**